app/src/main/res/values/dimens.xml

```diff
@@ -96,6 +96,23 @@
     <!-- Home Fragment -->
     <dimen name="home_fragment_top_toolbar_header_margin">60dp</dimen>
     <dimen name="home_collection_elevation">5dp</dimen>
+    <dimen name="home_recently_saved_padding_top">40dp</dimen>
+    <dimen name="home_recently_saved_padding_bottom">16dp</dimen>
+    <dimen name="home_recently_saved_padding_start">16dp</dimen>
+    <dimen name="home_recently_saved_padding_end">16dp</dimen>
+    <dimen name="home_show_all_padding_top">16dp</dimen>
+    <dimen name="home_show_all_padding_bottom">16dp</dimen>
+    <dimen name="recent_bookmark_item_height">128dp</dimen>
+    <dimen name="recent_bookmark_item_width">152dp</dimen>
+    <dimen name="recent_bookmark_item_padding">16dp</dimen>
+    <dimen name="recent_bookmark_item_card_radius">8dp</dimen>
+    <dimen name="recent_bookmark_item_card_elevation">1dp</dimen>
+    <dimen name="recent_bookmark_item_favicon_height">72dp</dimen>
+    <dimen name="recent_bookmark_item_favicon_corner_size">4dp</dimen>
+    <dimen name="recent_bookmark_item_title_height">30dp</dimen>
+    <dimen name="recent_bookmark_item_title_padding_top">7dp</dimen>
+    <dimen name="recent_bookmark_item_subtitle_height">16dp</dimen>
+    <dimen name="recent_bookmark_item_subtitle_margin_top">105dp</dimen>
```

Comment:
We can also create a new grouping `<!-- Recent bookmarks -->` to move towards a per feature component grouping.